src/digital_identity_service/share.session.result.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Validation = require('../yoti_common/validation');
+
+/**
+ * The share session result
+ *
+ * @class ShareSessionResult
+ */
+module.exports = class ShareSessionResult {
```

Comment:
What is the intent with this class regarding the result of a fetch session ?
Should there be different ? Probably yes to make things simpler, but if so, what name would we go with ?

Reply:
I also agree with different names like ShareSessionCreateResult and ShareSessionFetchResult OR CreateShareSessionResult and FetchShareSessionResult